evm-ds/src/precompiles/blake2.rs

```diff
@@ -0,0 +1,378 @@
+use std::borrow::Cow;
+
+use evm::executor::stack::{PrecompileFailure, PrecompileOutput};
+use evm::{Context, ExitError, ExitSucceed};
+
+/// Blake2 constants.
+mod consts {
+    pub(super) const INPUT_LENGTH: usize = 213;
+
+    /// The precomputed SIGMA.
+    ///
+    /// See [RFC 7693](https://datatracker.ietf.org/doc/html/rfc7693#section-2.7) specification for more details.
+    pub(super) const SIGMA: [[usize; 16]; 10] = [
```

Comment:
If it is borrowed from Aurora/Near, let's place an attribution at the top, please.